__tests__/presentation.test.js

```diff
@@ -363,9 +359,16 @@ function prepareTmpDir(dir) {
 }
 
 function emptyDir(dir) {
-    for (const file of fs.readdirSync(dir)) {
-        fs.unlink(path.join(dir, file), err => {
-            if (err) throw err;
-        });
+    let testFiles = [
+        `${tmpDir}/presentation-existing-rewrite.pptx`,
+        `${tmpDir}/presentation-existing-add-image.pptx`,
```

Comment:
why would you just not delete all files in this directory instead of manually specifying them?

Reply:
@heavysixer Because when you run the entire test suite (i.e. just "yarn test"), all files which are generated before presentation.test are inadvertently removed. Typically presentation.test is run as the first one anyway (but not always), so that's why we never noticed it before.